example-compose-files/sq-dce-postgres/docker-compose.yml

```diff
@@ -39,6 +39,7 @@ services:
       - sonarqube_extensions:/opt/sonarqube/extensions
       - sonarqube_logs:/opt/sonarqube/logs
       - sonarqube_temp:/opt/sonarqube/temp
+      - /opt/sonarqube/data
```

Comment:
I don't get this. Is that intended?

Reply:
Yes, in order to have one volume per replica the only solution is to use unnamed volume, which to me is completely fine as we are not binding them locally.

The other solution would have been to split the 3 replicas into 3 separate service like for the search nodes

Reply:
good point, thanks for clarifying!